operations/research-and-development/product/development-process/secure-software-development.md

```diff
@@ -0,0 +1,85 @@
+# Secure Software Development at Mattermost
+
+This document provides guidelines to secure software development at Mattermost. The document is at the moment a [1% draft](/company/about-mattermost/mindsets#drafts-at-1-50-99) and not conclusive.
+
+## Document scope
+
+This document covers the secure software development practices on all source repositories used to create software products offered by Mattermost.
+
+The product security team is overall in charge of secure software development practices by providing help and guidance and performing various verification activities, including code reviews and testing. Development teams are encouraged to adopt the secure software development practices described here. Third-party plugin developers are also welcome to use this document for guidance.
+
+## Recommended reading
+
+* [OSSF Scorecard Action Check Documentation](https://github.com/ossf/scorecard/blob/main/docs/checks.md)
+* [Overview of Software Supply Chain Attacks](https://www.crowdstrike.com/cybersecurity-101/cyberattacks/supply-chain-attacks/) by [Crowdstrike](https://www.crowdstrike.com/)
```

Comment:
```suggestion
```

Let's get rid of this one. It's the same link that got replaced with an Ars Technica article in the `Dependency pinning` section. It's otherwise a resonable link but the way it's written is a bit hyperbolic ("EVERY BUSINESS", emphasis theirs) and the section on prevention and detection is just pushing products.